test/test_shared.cpp

```diff
@@ -585,6 +616,7 @@ TEST(Shared_1)
     }
 }
 
+
```

Comment:
It's not random, it adds 2 lines between each unit test like in the rest of the file. I agree that we shouldn't do formatting in unrelated PR's though - it's pretty annoying. But i'm gonna keep these anyway now that they're already made...